client/src/components/Graph/GraphToggles.jsx

```diff
@@ -1,7 +1,15 @@
-import { Checkbox, FormControlLabel, List, ListItem, ListItemText, Switch } from "@mui/material"
-
-
-
+import { Checkbox, FormControlLabel, List, ListItem, ListItemText, Switch, Grid, Button } from "@mui/material"
+import generateZip from "../../hooks/generateZip";
+
+let labels = ["BaMnO3", "CaMnO3", "Cubic-SrMnO3", "Hex-SMO", "Hex-YMnO3", "LaMnO3"]
+let labelState = new Map();
+for (let j = 0; j < labels.length; j++) {
+    if (j < 3) {
+        labelState.set(j, true);
+    } else {
+        labelState.set(j, false);
+    }
+}
```

Comment:
Let's not create a hard-coded map/hashtable outside of the function. We can change item states by useState function from react.